.github/workflows/cron.yml

```diff
@@ -71,6 +71,89 @@ jobs:
           tags: |
             lycheeorg/lychee:testing-${{ github.run_id }}-devtools
 
+  alpha:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Check if alpha exists
+        run: |
+          echo "EXISTS=$(git ls-remote --heads https://github.com/LycheeOrg/Lychee refs/heads/alpha | wc -l") >> $GITHUB_OUTPUT
```

Comment:
Isn't the whole point of this that alpha exists?

Reply:
yes, but I kinda wanted to have a safegard if the branch did not exists :)